tools/create-extension/_package.json

```diff
@@ -12,17 +12,17 @@
 		"lint:css": "wp-scripts lint-style",
 		"lint:js": "wp-scripts lint-js",
 		"lint:md:docs": "wp-scripts lint-md-docs",
-		"lint:md:js": "wp-scripts lint-md-js",
 		"lint:pkg-json": "wp-scripts lint-pkg-json",
 		"packages-update": "wp-scripts packages-update",
 		"start": "wp-scripts start",
 		"test:e2e": "wp-scripts test-e2e",
 		"test:unit": "wp-scripts test-unit-js"
 	},
 	"devDependencies": {
-		"@wordpress/scripts": "^12.2.1",
+		"@wordpress/scripts": "^26.10.0",
 		"@woocommerce/eslint-plugin": "latest",
-		"@woocommerce/dependency-extraction-webpack-plugin": "1.6.0"
+		"@woocommerce/dependency-extraction-webpack-plugin": "1.6.0",
+		"prettier": "^2.8.8"
```

Comment:
Need prettier as dep for the formatting to work properly, also it needs to be sub version 3 due to some issue with compatibility of our now very old eslint plugin (that plugin needs re-releasing at some point, its been a year since it was released last)